### PR TITLE
Add documentation for accessing codebuild logs

### PR DESCRIPTION
**Pull request by thk123**
The process is a little painful, but better than not being able to access at all.

I _think_ the picture will work when merged?

- [x] Each commit message has a non-empty body, explaining why the change was made.
- [x] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [x] The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [x] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [ ] My commit message includes data points confirming performance improvements (if claimed).
- [x] My PR is restricted to a single feature or bugfix.
- [x] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

**Review by danpoe — APPROVED** [review]

**Comment by hannes-steffenhagen-diffblue** [comment]
Here’s what this looks like when merged: https://github.com/thk123/cbmc/blob/document-log-access/AccessingCodebuildLogs.md (picture works, yes).

Quick question, what was the thought behind putting this into the main directory instead of doc?

File: AccessingCodebuildLogs.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+To get hold of the logs you need to do the following:
+
+1. Get the job URL
+
+Right click on the relevant job, and copy the link URL
+
+![GetLink](doc/assets/GetLink.png)
+
+2. Extrac the ID
```
Comment (danpoe) [inline AccessingCodebuildLogs.md]: Extrac -> Extract

**Review by danpoe — APPROVED** [review]

**Comment by thk123** [comment]
@hannes-steffenhagen-diffblue thought was it is aimed at contributors so they don't want to dig through docs (I'd imagine it would really live inside a `CONTRIBUTORS.md` but perhaps better in docs - let me know if you'd rather there and I'll move it.

**Review by NlightNFotis — APPROVED** [review]
LGTM

**Review by thomasspriggs — APPROVED** [review]

**Review by hannes-steffenhagen-diffblue — APPROVED** [review]

**Review by martin-cs — APPROVED** [review]
I have tried this as a contributor with no special access permissions and it seems to work.

**Review by peterschrammel — APPROVED** [review]

**Pull request merged**